from PyQt5 import QtCore, QtGui, QtWidgets

class Ui_categoryAdd(object):

    def setupUi(self, categoryAdd):
        categoryAdd.setObjectName("categoryAdd")
        categoryAdd.setFixedSize(950, 525)
        self.logo = QtWidgets.QLabel(categoryAdd)
        self.logo.setGeometry(QtCore.QRect(50, 15, 1000, 100))
        self.logo.setPixmap(QtGui.QPixmap('WOJlogo.png'))
        self.logo.setObjectName("logo")

        self.category = QtWidgets.QLineEdit(categoryAdd)
        self.category.setGeometry(QtCore.QRect(250, 120, 640, 25))
        self.category.setObjectName("category")
        self.category_label = QtWidgets.QLabel(categoryAdd)
        self.category_label.setGeometry(QtCore.QRect(100, 120, 150, 25))
        self.category_label.setObjectName("category_label")

        self.question1 = QtWidgets.QLineEdit(categoryAdd)
        self.question1.setGeometry(QtCore.QRect(250, 150, 640, 25))
        self.question1.setObjectName("question1")
        self.q1_label = QtWidgets.QLabel(categoryAdd)
        self.q1_label.setGeometry(QtCore.QRect(100, 150, 150, 25))
        self.q1_label.setObjectName("q1_label")
        self.answer1 = QtWidgets.QLineEdit(categoryAdd)
        self.answer1.setGeometry(QtCore.QRect(250, 180, 640, 25))
        self.answer1.setObjectName("answer1")
        self.a1_label = QtWidgets.QLabel(categoryAdd)
        self.a1_label.setGeometry(QtCore.QRect(100, 180, 150, 25))
        self.a1_label.setObjectName("a1_label")

        self.question2 = QtWidgets.QLineEdit(categoryAdd)
        self.question2.setGeometry(QtCore.QRect(250, 210, 640, 25))
        self.question2.setObjectName("question2")
        self.q2_label = QtWidgets.QLabel(categoryAdd)
        self.q2_label.setGeometry(QtCore.QRect(100, 210, 150, 25))
        self.q2_label.setObjectName("q2_label")
        self.answer2 = QtWidgets.QLineEdit(categoryAdd)
        self.answer2.setGeometry(QtCore.QRect(250, 240, 640, 25))
        self.answer2.setObjectName("answer2")
        self.a2_label = QtWidgets.QLabel(categoryAdd)
        self.a2_label.setGeometry(QtCore.QRect(100, 240, 150, 25))
        self.a2_label.setObjectName("a2_label")

        self.question3 = QtWidgets.QLineEdit(categoryAdd)
        self.question3.setGeometry(QtCore.QRect(250, 270, 640, 25))
        self.question3.setObjectName("question3")
        self.q3_label = QtWidgets.QLabel(categoryAdd)
        self.q3_label.setGeometry(QtCore.QRect(100, 270, 150, 25))
        self.q3_label.setObjectName("q3_label")
        self.answer3 = QtWidgets.QLineEdit(categoryAdd)
        self.answer3.setGeometry(QtCore.QRect(250, 300, 640, 25))
        self.answer3.setObjectName("answer3")
        self.a3_label = QtWidgets.QLabel(categoryAdd)
        self.a3_label.setGeometry(QtCore.QRect(100, 300, 150, 25))
        self.a3_label.setObjectName("a3_label")

        self.question4 = QtWidgets.QLineEdit(categoryAdd)
        self.question4.setGeometry(QtCore.QRect(250, 330, 640, 25))
        self.question4.setObjectName("question4")
        self.q4_label = QtWidgets.QLabel(categoryAdd)
        self.q4_label.setGeometry(QtCore.QRect(100, 330, 150, 25))
        self.q4_label.setObjectName("q4_label")
        self.answer4 = QtWidgets.QLineEdit(categoryAdd)
        self.answer4.setGeometry(QtCore.QRect(250, 360, 640, 25))
        self.answer4.setObjectName("answer4")
        self.a4_label = QtWidgets.QLabel(categoryAdd)
        self.a4_label.setGeometry(QtCore.QRect(100, 360, 150, 25))
        self.a4_label.setObjectName("a4_label")

        self.question5 = QtWidgets.QLineEdit(categoryAdd)
        self.question5.setGeometry(QtCore.QRect(250, 390, 640, 25))
        self.question5.setObjectName("question5")
        self.q5_label = QtWidgets.QLabel(categoryAdd)
        self.q5_label.setGeometry(QtCore.QRect(100, 390, 150, 25))
        self.q5_label.setObjectName("q5_label")
        self.answer5 = QtWidgets.QLineEdit(categoryAdd)
        self.answer5.setGeometry(QtCore.QRect(250, 420, 640, 25))
        self.answer5.setObjectName("answer5")
        self.a5_label = QtWidgets.QLabel(categoryAdd)
        self.a5_label.setGeometry(QtCore.QRect(100, 420, 150, 25))
        self.a5_label.setObjectName("a5_label")

        self.save = QtWidgets.QPushButton(categoryAdd)
        self.save.setGeometry(QtCore.QRect(260, 460, 211, 51))
        self.save.setObjectName("save")
        self.cancel = QtWidgets.QPushButton(categoryAdd)
        self.cancel.setGeometry(QtCore.QRect(510, 460, 211, 51))
        self.cancel.setObjectName("cancel")

        self.retranslateUi(categoryAdd)
        QtCore.QMetaObject.connectSlotsByName(categoryAdd)

    def retranslateUi(self, categoryAdd):
        _translate = QtCore.QCoreApplication.translate
        categoryAdd.setWindowTitle(_translate("categoryAdd", "Add Category"))
        self.category_label.setText(_translate("categoryAdd", "Category Name"))
        self.q1_label.setText(_translate("categoryAdd", "Question #1"))
        self.a1_label.setText(_translate("categoryAdd", "Answer #1"))
        self.a2_label.setText(_translate("categoryAdd", "Answer #2"))
        self.q2_label.setText(_translate("categoryAdd", "Question #2"))
        self.a3_label.setText(_translate("categoryAdd", "Answer #3"))
        self.q3_label.setText(_translate("categoryAdd", "Question #3"))
        self.a4_label.setText(_translate("categoryAdd", "Answer #4"))
        self.q4_label.setText(_translate("categoryAdd", "Question #4"))
        self.a5_label.setText(_translate("categoryAdd", "Answer #5"))
        self.q5_label.setText(_translate("categoryAdd", "Question #5"))
        self.save.setText(_translate("categoryAdd", "Save"))
        self.cancel.setText(_translate("categoryAdd", "Cancel"))